colin-api/src/colin_api/resources/v1/corporations.py

```diff
@@ -177,6 +215,13 @@ def build_nr_sql(corp_num_sql):
                    from bc_registries.corp_nr_num_vw \
                    where corp_num = {corp_num_sql};'
 
+    @staticmethod
+    def build_incorp_info_sql(corp_num_sql):
+        """Build business info sql."""
+        return f'select * \
+                   from bc_registries.solr_dataimport_conflicts_vw \
+                   where id = {corp_num_sql};'
```

Comment:
is this query right?  i.e. is this the right view we should be querying?

Reply:
Yes

Reply:
This view represents seems to contain businesses that have solr data import conflicts.  

I don't think it will return results for all the businesses that actually exist in COLIN.

Reply:
Make sense